dev-python/uri_template/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@chymera.eu</email>
+		<name>Horea Christian</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+		<name>Python</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="gitlab">gitlab.linss.com/open-source/uri_template/</remote-id>
+	</upstream>
```

Comment:
`remote-id type="gitlab"` is for gitlab.com. I don't think there's a way to refer to someone else's GitLab instance here :/

```suggestion
```

Reply:
Can confirm it won't work

Reply:
Can confirm it won't work

Reply:
@MatthewGentoo 

```
chymera@decohost ~/src/gentoo $ ag "gitlab\." */*/metadata.xml | ag remote
app-backup/deja-dup/metadata.xml:28:		<remote-id type="gitlab">https://gitlab.gnome.org/World/deja-dup</remote-id>
app-crypt/adcli/metadata.xml:13:    <remote-id type="gitlab">https://gitlab.freedesktop.org/realmd/adcli</remote-id>
dev-python/uri_template/metadata.xml:13:		<remote-id type="gitlab">gitlab.linss.com/open-source/uri_template/</remote-id>
net-libs/libslirp/metadata.xml:9:		<remote-id type="gitlab">gitlab.freedesktop.org/slirp/libslirp</remote-id>
sci-libs/symmetrica/metadata.xml:22:    <remote-id type="gitlab">https://gitlab.com/sagemath/symmetrica</remote-id>
```

It seems to be used this way. How should I refer to upstream then? They have a mirror on centralized GitHub.

Reply:
ok, just removed the upstream then..